rules/aws_cloudtrail_rules/aws_cloudtrail_account_discovery.py

```diff
@@ -0,0 +1,21 @@
+from panther_base_helpers import deep_get
+
+DISCOVERY_EVENTS = [
+    "GetAlternateContact",
+    "GetContactInformation",
+    "PutAlternateContact",
+    "PutContactInformation",
+    "DescribeAccount",
```

Comment:
Should we also add "DescribeAccount", "DescribeOrganization"?

Reply:
We were looking [here](https://docs.aws.amazon.com/accounts/latest/reference/API_Operations.html) for the Account Management API event names 

I would be happy to add "DescribeOrganization" and "DescribeOrganizationUnit" if I missed them! Just let me know one way or the other.

Reply:
After looking at the description for T1087, I'm wondering if this capture's the intent of the technique. This sub-technique may be more relevant https://attack.mitre.org/techniques/T1087/004/ 
Which should look for: 
eventName:ListUsers, ListRoles
eventSource:iam.amazonaws.com

Reply:
I was going off what was in the description of the Asana task [here](https://app.asana.com/0/1202920465233238/1203074962567700/f)

Reply:
@edyesed thoughts? I think from a perspective of an attacker there isn't much benefit of going for the contact information. The technique seems intended to capture account reconnaissance from things like ListUsers and ListRoles

Reply:
I concur that the sub technique as listed on mitre's website does focus on IAM users. As I recall, we've got detections for that particular bit of recon. I think this detection is relevant in its current state, and that ATT&CK will eventually grow to recognize the complexity of groups of cloud accounts

Reply:
This doesn't need to be blocking for this particular detection, and I think, in general, we should combine eventSource and eventName. Here's all of the potential eventNames that would match this list. Notice that DescribeAccount occurs in two different eventSources, and each of those sources is separate from the first batch. 


```
account:GetAlternateContact
account:GetContactInformation
account:PutAlternateContact
account:PutContactInformation
organizations:DescribeAccount
workspaces:DescribeAccount
```